components/jeune/BlocInformationJeune.tsx

```diff
@@ -158,10 +166,10 @@ function DossierExterne(props: { href: string; onClick: () => void }) {
   )
 }
 
-function LienVersHistorique(props: { idJeune: string }) {
+function LienVersHistorique(props: { idJeune: string; pathPrefix: string }) {
   return (
     <Link
-      href={`/mes-jeunes/${props.idJeune}/historique`}
+      href={`${props.pathPrefix}/${props.idJeune}/historique`}
```

Comment:
```suggestion
      href={`${pathPrefix}/${idJeune}/historique`}
```

destructuring, retirer les props